test/mitmproxy/tservers.py

```diff
@@ -122,6 +122,7 @@ def get_proxy_config(cls):
             cadir = cls.cadir,
             authenticator = cls.authenticator,
             add_upstream_certs_to_client_chain = cls.add_upstream_certs_to_client_chain,
+            ssl_insecure = True,
```

Comment:
Actually, HTTP/2 misses this value - because it has its own TestBase.
@s4chin please add the same option here: https://github.com/mitmproxy/mitmproxy/blob/5d0de16f57e8af971fd282660e3c32d08e7e1317/test/mitmproxy/test_protocol_http2.py#L97-L101